### PR TITLE
fix: get_conf_option("modules") returns "", do not check against None

### LINKED ISSUE #677: bug introduced by commit c4276791713e47a76433122ddf7fdedb2a5a3d44 (pytest <=7.4.4)
I am really sorry to say, a subtle bug has been introduced by this commit, and I didn't see it while doing the review 😫 

At the moment, `modules` is always `['']` which prevents Redis to start...
I wonder how it could pass all tests ?

In anyways, the fix is super easy, in file `pytest_redis/config.py` instead of:

```
if (modules := get_conf_option("modules")) is not None:
                                             ^^^^^^^^^^
        modules = modules.split(",")
else:
        modules = []
```

The config key is empty string so it is enough to make the test as: `if (modules := get_conf_option("modules")):`

Again, I apologize for not having seen it before.

### PR DESCRIPTION
**Pull request by mguijarr**
Fix #677

### CODE REVIEW

**Comment by fizyk** [comment]
@mguijarr are you able to provide a bugfix newsfragment?

**Comment by mguijarr** [comment]
Done! I added the fragment. Sorry for the delay

File: pytest_redis/config.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def get_conf_option(option: str) -> Any:
         return request.config.getoption(option_name) or request.config.getini(option_name)
 
     port = get_conf_option("port")
-    if (modules := get_conf_option("modules")) is not None:
+    if (modules := get_conf_option("modules")):
```
Comment (fizyk) [inline pytest_redis/config.py:36]: the parenthesis left at the moment is what black complains about :)

Comment (mguijarr) [reply pytest_redis/config.py:36]: 🤦🏻‍♂️ sorry

Comment (mguijarr) [reply pytest_redis/config.py]: Should be fine now

**Comment by fizyk** [comment]
Thank you @mguijarr,  release is in progress :)

**Pull request merged**